nbgrader/apps/feedbackapp.py

```diff
@@ -37,38 +26,74 @@ class FeedbackApp(BaseNbConvertApp):
 
     examples = Unicode(dedent(
```

Comment:
Ah, good point about it not needing to be dedented. I'll see what happens if I also don't make it a traitlet.